OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -77,24 +78,8 @@ public Move(Actor self, CPos destination, SubCell subCell, WDist nearEnough)
 			this.nearEnough = nearEnough;
 		}
 
-		public Move(Actor self, CPos destination, Actor ignoredActor)
-		{
-			mobile = self.Trait<Mobile>();
-
-			getPath = () =>
-			{
-				List<CPos> path;
-				using (var search =
-					PathSearch.FromPoint(self.World, mobile.Info, self, mobile.ToCell, destination, false)
-					.WithIgnoredActor(ignoredActor))
-					path = self.World.WorldActor.Trait<IPathFinder>().FindPath(search);
-				return path;
-			};
-
-			this.destination = destination;
-			nearEnough = WDist.Zero;
-			this.ignoredActor = ignoredActor;
-		}
+		public Move(Actor self, CPos destination, WDist nearEnough)
```

Comment:
Please set the default value for `ignoredActor` to null in the overload above instead of duplicating this.